.github/run-clang-format.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python
+
+'''Run clang-format on files in this repository
+
+By default, runs on all files, but may pass specific files.
+'''
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import argparse
+import inspect
+import fnmatch
+import io
+import os
+import re
+import stat
+import subprocess
+import sys
+import time
+
+
+THIS_FILE = os.path.normpath(os.path.abspath(inspect.getsourcefile(lambda: None)))
+THIS_DIR = os.path.dirname(THIS_FILE)
+# THIS_DIR = REPO_ROOT/tests/bin
```

Comment:
You will need to adjust this to the new location of the file

Reply:
Done!